src/lib/utils.ts

```diff
@@ -39,6 +39,9 @@ export function partitionCompare<T>(first: T[], second: T[]) {
   }
 }
 
+export const middleEllipsis = (value: string): string =>
```

Comment:
common middle ellipsis helper for arbitrary strings

Reply:
@reallybeard FYI also ^^